docs/iris/src/whatsnew/latest.rst

```diff
@@ -140,7 +140,12 @@ This document explains the changes made to Iris for this release
 
 * `@stephenworsley`_ changed Iris objects loaded from NetCDF-CF files to have
   ``units='unknown'`` where the corresponding NetCDF variable has no ``units``
-  property. Previously these cases defaulted to ``units='1'``. (:pull:`3795`)
+  property. Previously these cases defaulted to ``units='1'``. (:pull:`3795`).
+  This affects loading of coordinates whose file variable has no "units"
+  attribute (not valid, under `CF units rules`_):  These will now have units
+  of `"unknown"`, rather than `"1"`, which **may prevent the creation of
+  a hybrid vertical coordinate**.  While these cases used to "work", this was
+  never really correct behaviour.
```

Comment:
```suggestion
  property. Previously these cases defaulted to ``units='1'``.
  This affects loading of coordinates whose file variable has no "units"
  attribute (not valid, under `CF units rules`_):  These will now have units
  of `"unknown"`, rather than `"1"`, which **may prevent the creation of
  a hybrid vertical coordinate**.  While these cases used to "work", this was
  never really correct behaviour. (:pull:`3795`)
```

In accordance with the order in [Contributing a “What’s New” entry](https://scitools-iris.readthedocs.io/en/latest/developers_guide/documenting/whats_new_contributions.html#writing-a-contribution)